VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitVideoFragment.java

```diff
@@ -799,10 +798,9 @@ public void onConfigurationChanged(Configuration newConfig) {
         //TODO - should we use load different layout file?
         if (getResources().getConfiguration().orientation == Configuration.ORIENTATION_LANDSCAPE) {
             messageContainer.setVisibility(View.GONE);
-            if (Build.VERSION.SDK_INT < 16) {
-                getActivity().getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,
+            getActivity().getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,
                     WindowManager.LayoutParams.FLAG_FULLSCREEN);
-            } else {
+            if (Build.VERSION.SDK_INT >= 16) {
                 View decorView = getActivity().getWindow().getDecorView();
                 int uiOptions = View.SYSTEM_UI_FLAG_FULLSCREEN;
                 decorView.setSystemUiVisibility(uiOptions);
```

Comment:
I think we can remove this, and rely on the window flag. The documentation states:

> This approach to going fullscreen is best used over the window flag when it is a transient state -- that is, the application does this at certain points in its user interaction where it wants to allow the user to focus on content, but not as a continuous state. For situations where the application would like to simply stay full screen the entire time (such as a game that wants to take over the screen), the [window flag][] is usually a better approach. The state set here will be removed by the system in various situations (such as the user moving to another application) like the other system UI states.

[window flag]: https://developer.android.com/reference/android/view/WindowManager.LayoutParams.html#FLAG_FULLSCREEN